docs/amber_script.md

```diff
@@ -446,6 +447,10 @@ EXPECT {buffer_name} IDX _x_in_pixels_ _y_in_pixels_ \
 
 # Checks that |buffer_1| contents are equal to those of |buffer_2|
 EXPECT {buffer_1} EQ_BUFFER {buffer_2}
+
+# Checks that the Root Mean Square Error when comparing |buffer_1| to
+# |buffer_2| is less than or equal too |tolerance|
```

Comment:
The syntax looks ok to me.  I'd clarify that |tolerance| is unitless.
E.g. when one buffer is an unbiased estimater of the other, then RMSE is the standard deviation, which is unitless.